src/utilities/jwtUtilities.js

```diff
@@ -2,11 +2,16 @@ const jwt = require('jsonwebtoken');
 const fs = require('fs');
 
 let key = "";
-fs.readFile('../key.txt', (err, data) => {
+fs.readFile('./key.txt', (err, data) => {
```

Comment:
because this is async, it is causing a race condition where `createToken` can get called before the file is done reading. Probably best to make it Synchronous to make sure key is set before jwt.sign is called